lib/command/run.rb

```diff
@@ -84,19 +87,25 @@ class Run < Base # rubocop:disable Metrics/ClassLength
       ```
     EX
 
+    DEFAULT_JOB_CPU = "1"
+    DEFAULT_JOB_MEMORY = "2Gi"
     MAGIC_END = "---cpl run command finished---"
 
     attr_reader :interactive, :detached, :location, :original_workload, :runner_workload,
+                :default_image, :default_cpu, :default_memory,
                 :container, :expected_deployed_version, :job, :replica, :command
 
-    def call # rubocop:disable Metrics/MethodLength
+    def call # rubocop:disable Metrics/CyclomaticComplexity, Metrics/MethodLength, Metrics/PerceivedComplexity
       @interactive = config.options[:interactive] || interactive_command?
       @detached = config.options[:detached]
       @log_method = config.options[:log_method]
 
       @location = config.location
       @original_workload = config.options[:workload] || config[:one_off_workload]
       @runner_workload = "#{original_workload}-runner"
+      @default_image = cp.latest_image_from([], app_name: config.app)
```

Comment:
I guess default image should be the current image of the app and not the latest? I mean if we want latest we do `--latest` which is easy. How you do the current one? `--current`? I mean surely one can specify exact, but this is complex.

Anyway this may be a breaking change.

Reply:
This is actually not the latest image, and it has nothing to do with which image is used by the jobs. Passing an empty array to `cp.latest_image_from` simply returns `{APP_NAME}:NO_IMAGE_AVAILABLE`:

https://github.com/shakacode/control-plane-flow/blob/2133ff4be886d978b1becec7b7f9ae8c4943fb4a/lib/core/controlplane.rb#L65-L71

I just wanted to reset the runner workload image, because we won't be updating it anymore. If we were to set it to anything else, it would eventually become outdated, so that's why I thought `NO_IMAGE_AVAILABLE` would be better.

Images are now set for each job separately, in `build_job_start_yaml`. And there, we have the correct logic for defaulting to the current image:

https://github.com/shakacode/control-plane-flow/blob/2133ff4be886d978b1becec7b7f9ae8c4943fb4a/lib/command/run.rb#L345-L353

Perhaps I can rename the `@default_image` variable to make it less confusing? It's meant to be "default image for workload", not "default image for jobs". So maybe `@workload_default_image`?

Reply:
Yes, `latest` word is confusing as it has different meaning as term (please change it). Overall, I guess I more like `NO_IMAGE` way for the runner. And set image per job.

Reply:
Ok, done in 1c6e103baff6c972e3a2db9b574552f6e23fd27d.